interfaces/automated-testing/rid/injection.yaml

```diff
@@ -0,0 +1,159 @@
+openapi: 3.0.2
+info:
+  title: Remote ID Test Data Injection
+  version: 0.0.1
+  description: >-
+    This interface is implemented by every Service Provider wishing to be tested by the automated testing framework.
+    The automated testing suite calls this interface to inject flight-related test data into the Service Provider system under test.
```

Comment:
This seems pretty specific.  Is there a design doc that this was covered in?  I would like to review what the overall outline for the approach for Test Data Injection is before we define the schema for it.

Reply:
Implied approach added to [README](https://github.com/BenjaminPelletier/dss/tree/automatedtesting/api/interfaces/automated-testing/rid).